sklearn/tests/test_metaestimators_metadata_routing.py

```diff
@@ -287,6 +287,14 @@ def enable_slep006():
         "cv_name": "cv",
         "cv_routing_methods": ["fit"],
     },
+    {
+        "metaestimator": IterativeImputer,
+        "estimator_name": "estimator",
+        "estimator": ConsumingRegressor,
+        "X": X,
+        "y": y,
```

Comment:
Can you also add `skip_complete=False` to make sure we train models even when no missing values are present? Default is False, but better to be sure.